packages/integrations/sitemap/test/fixtures/static/src/pages/endpoint.json.ts

```diff
@@ -1,8 +1,8 @@
 export async function GET({}) {
-  return {
-    body: JSON.stringify({
-      name: 'Astro',
-      url: 'https://astro.build/',
-    }),
-  };
+	return Response.json(
+		JSON.stringify({
+			name: 'Astro',
+			url: 'https://astro.build/',
+		})
+	);
```

Comment:
```suggestion
	return Response.json({
		name: 'Astro',
		url: 'https://astro.build/',
	});
```

You can pass the object directly for `Response.json`